packages/types/src/indexed-db.ts

```diff
@@ -122,9 +123,9 @@ export interface PenumbraDb extends DBSchema {
     key: 'params';
     value: Jsonified<FmdParameters>;
   };
-  TRANSACTION_INFO: {
-    key: Jsonified<Required<TransactionInfo>['id']['inner']>; // base64
-    value: Jsonified<TransactionInfo>;
+  TRANSACTIONS: {
```

Comment:
Table renamed to better display so that we no longer store the full TransactionInfo